plugins/shovel.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: shovel
+spec:
+  homepage: https://github.com/dodopizza/kubectl-shovel
+  shortDescription: Generate CPU shovel graphs from pods
+  version: v0.1.5
+  description: |
+    Plugin for gathering diagnostics from running in k8s dotnet applications
+  caveats: |
+    It can work with .Net Core 3.0+ applications and Kubernetes clusters with docker runtime
```

Comment:
can you line wrap at 80 chars

Reply:
also I think the right spelling is ".NET Core"

Reply:
fixed